test/controllers/lists_controller_test.rb

```diff
@@ -0,0 +1,24 @@
+require 'test_helper'
```

Comment:
Delete the `blocitoff/test` directory as we're using `blocitoff/spec` to house our tests.

Consider using the `-T` option with `rails new ...` to skip creating the default `test/` directory on future projects, when you know that you're going to test with RSpec. e.g. `rails new mynewproject -T`

For a reminder of `rails new` options like `-T`, run `rails new --help`